packages/dapp-svelte-wallet/react-ui/src/components/tests/WalletConnection.test.js

```diff
@@ -0,0 +1,145 @@
+import { act } from '@testing-library/react';
+import { mount } from 'enzyme';
+import { ApplicationContext } from '../../contexts/Application';
+import WalletConnection from '../WalletConnection';
+
+jest.mock('@agoric/eventual-send', () => ({
```

Comment:
Mocking eventual-send seems like the right thing to do for unit tests, since it's analogous to mocking `fetch` for remote requests. I did try to call lockdown, but there was an issue I ran into with `jest` (I tried invoking `lockdown` before and after `jest` initializes and got the same error and the tests did not run):

```
Unexpected intrinsic intrinsics.ArrayBuffer.__proto__ at %FunctionPrototype%
```